nevergrad/benchmark/test_experiments.py

```diff
@@ -32,6 +32,10 @@ def test_experiments_registry(name: str, maker: tp.Callable[[], tp.Iterator[expe
     if "_pgan" in name and os.environ.get("CIRCLECI", False):
         raise SkipTest("Too slow in CircleCI")
 
+    # CompilerGym sends http requests that CircleCI does not like.
+    if "compiler" in name and os.environ.get("CIRCLECI", False):
+        raise SkipTest("No HTTP request in CircleCI")
```

Comment:
please put this in the code of the function that does not work instead (if CIRCLECI then raise SkipTest). Relying on the name is never a good idea